worker/wallet.js

```diff
@@ -105,13 +235,7 @@ export async function checkWithdrawal ({ data: { id, hash }, boss, models, lnd }
       status = 'PATHFINDING_TIMEOUT'
     } else if (wdrwl?.failed.is_route_not_found) {
       status = 'ROUTE_NOT_FOUND'
-    }
-    await serialize(models, models.$executeRaw`
-      SELECT reverse_withdrawl(${id}::INTEGER, ${status}::"WithdrawlStatus")`)
-  } else {
-    // we need to requeue to check again in 5 seconds
-    const startAfter = new Date(wdrwl.created_at) > datePivot(new Date(), { minutes: -5 }) ? 5 : 60
-    await boss.send('checkWithdrawal', { id, hash }, { ...walletOptions, startAfter })
+    } else await serialize(models, models.$executeRaw`SELECT reverse_withdrawl(${id}::INTEGER, ${status}::"WithdrawlStatus")`)
```

Comment:
I think this means we aren't reversing withdrawals if we know the status. This looks like a typ.

Reply:
This was a typo. I'm working on this locally. I'll push up in a bit.

Reply:
Ah, yes. Didn't read the code properly, just thought I deleted an existing `else` around `reverse_withdrawal` on accident. But I deleted the `else` for requeueing which was correct.